src/scenarios/shared/util.py

```diff
@@ -64,4 +70,13 @@ def getruntimeidentifier():
 
     return rid
 
+def pythoncommand():
+    if sys.platform == 'win32':
+        return 'py'
+    else:
+        return 'python3'
+
+def iswin():
+    return sys.platform == 'win32'
```

Comment:
If you're adding this here why not use it above in `pythoncommand` and other places?

Reply:
Will replace other ifs with this method